hack/test-integration.sh

```diff
@@ -55,7 +55,7 @@ runTests() {
 trap cleanup EXIT
 
 # Convert the CSV to an array of API versions to test
-IFS=',' read -a apiVersions <<< ${KUBE_TEST_API_VERSIONS}
+IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
```

Comment:
Yes. That works as well, but am wondering if its better.
I found that using the read command and using Parameter Expansion are the 2 most popular ways of converting a CSV to an array: http://stackoverflow.com/questions/918886/how-do-i-split-a-string-on-a-delimiter-in-bash.
But it seems people prefer IFS since Parameter expansion breaks if the array values contain "*" or any other special regex wildcard.
In our simple case, both work, but am wondering if you have a preference for one over the other?